### PR TITLE
docs(proposals): add How? for pipeline info admin endpoint (#796)

### PR DESCRIPTION
**Pull request by henschwartz**
## Summary
- Adds the `## How?` section to `docs/proposals/00796_pipeline-info-admin-endpoint.md` after What?/Why? (#823) landed.
- Captures Tim’s review Decisions as an implementation plan: `GET /api/pipelines` on existing `PingoraAdminService`, `serde_json`, loopback/`allow_public_admin`, `ListenerPipelines` as live source, `chain_names[]`, empty `filters: []` / `filter_count: 0`, and cheap per-listener queries.
- Describes required `FilterPipeline` introspection, example JSON shape, reload edge cases, and unit/integration test plan for a future implementing PR.

## Test plan
- [ ] Confirm proposal still matches #823 Decisions (authz, empty chains, per-listener, admin exclusion)
- [ ] Confirm How? does not claim an implementing PR yet (placeholder until code lands)
- [ ] Confirm CI `proposal-check` allows How? on this follow-up (file is modified, not newly added)

Related: #796, #793 (`praxisctl pipelines` consumer later).

### CODE REVIEW

File: docs/proposals/00796_pipeline-info-admin-endpoint.md [hunk 1]
```diff
@@ -157,12 +157,107 @@ Record of answers that close the earlier open questions from
...
+automatically.
+
+**Introspection.** `FilterPipeline` today does not expose its
+ordered filters publicly. Add a dedicated public snapshot API on
+the filter crate (for example `FilterPipeline::introspection()`)
+that walks `PipelineFilter` / `ResolvedBranch` and returns:
+
+- filter type (`AnyFilter::name()`), optional user `name`
+- request/response conditions and `failure_mode`
+- per-filter body access/mode (from `HttpFilter` trait methods)
```
[the remark this answers is not in the document]
Comment (henschwartz) [reply docs/proposals/00796_pipeline-info-admin-endpoint.md]: done

File: docs/proposals/00796_pipeline-info-admin-endpoint.md [hunk 2]
```diff
@@ -157,12 +157,107 @@ Record of answers that close the earlier open questions from
...
+```json
+{
+  "listeners": [
+    {
+      "name": "web",
+      "address": "0.0.0.0:8080",
+      "protocol": "http",
+      "tls": false,
+      "chain_names": ["main"],
+      "filter_count": 2,
```
[the remark this answers is not in the document]
Comment (henschwartz) [reply docs/proposals/00796_pipeline-info-admin-endpoint.md]: done

File: docs/proposals/00796_pipeline-info-admin-endpoint.md [hunk 3]
```diff
@@ -157,12 +157,107 @@ Record of answers that close the earlier open questions from
...
+      "filters": [
+        {
+          "index": 0,
+          "filter": "router",
+          "name": null,
+          "conditions": [],
+          "response_conditions": [],
+          "failure_mode": "closed",
+          "phases": ["request"],
```
[the remark this answers is not in the document]
Comment (henschwartz) [reply docs/proposals/00796_pipeline-info-admin-endpoint.md:256]: done

File: docs/proposals/00796_pipeline-info-admin-endpoint.md [hunk 4]
```diff
@@ -157,12 +157,107 @@ Record of answers that close the earlier open questions from
...
+          "response_body": { "access": "none", "mode": "stream" },
+          "branches": []
+        }
+      ]
+    }
+  ]
+}
+```
+
+Per-listener queries return a single `listener` object (or 404)
```
[the remark this answers is not in the document]
Comment (henschwartz) [reply docs/proposals/00796_pipeline-info-admin-endpoint.md]: done

**Review by shaneutt — APPROVED** [review]
I dig it. Let's try it.
